### ISSUE TITLE
Cannot install iOS app

### ISSUE THREAD

**Issue by pgorzelany-objectivity** [microsoft/appcenter#813]
**What App Center service does this affect?**
iOS app distribution

**Describe the bug**
All devices are provisioned (green) and a release has been prepared but the testers cannot install the build.
There is no install button available for the testers, only a "GET" button. Once the tester clicks GET he gets an error "Oops,there was a problem building the app".
Automatic device management is turned on.

**To Reproduce**
1. As a tester select a release
2. Click GET

**Expected behavior**
The app should be downloaded on the device

**Screenshots**
![AppCenterError](https://user-images.githubusercontent.com/39732840/62372029-aa356c00-b536-11e9-9cfc-30b3bf45dee3.png)

**iPad (please complete the following information):**
 - Device: iPad
 - OS: iOS 12.4
 - Browser Safari

**Comment by weslley39** [comment]
Same issue here 😞

**Comment by weslley39** [comment]
_The problem was that Apple updated the format of their UDIDs from their previous generations iPhones to the iPhone XS._ 
This issue is only happening on iPhone XR for me.

**Comment by lumaxis** [comment]
We believe we have identified the cause of this issue and are working on a fix that should be available within the next few hours 🙏🏼

**Comment by pgorzelany-objectivity** [comment]
@lumaxis thanks for the quick response. Our testers are now able to install the app :)

**Comment by lumaxis** [comment]
Cool, thanks for confirming @pgorzelany-objectivity ! I'll close this Issue.

@weslley39 If you're still experiencing a problem, please reply here or open a new Issue with details.

**Comment by tomduncalf** [comment]
Hey @lumaxis, I am seeing this issue on an iPhone XR and two iPad Pros, both of which have the new style UDIDs (they are first devices with this style UDID I have tried). As you say the issue should be fixed, is there any way I can get more debug info?

**Comment by klogeaage** [comment]
Hi @lumaxis, could this have been resurfacing recently? I have two testers that can no longer install my app on their XS and X phones, while the rest with older devices have no problems. The testers with X-devices used to be able to do so, but now get the "Oops! There was a problem..." message. Since they are unable to start the app, I really don't have much more information.

**Comment by techericb** [comment]
Hi @klogeaage, did you ever find a resolution?  I am seeing the same issue with my testers now (the latest tester is using XS.

**Comment by klogeaage** [comment]
Yes I did. The problem was that my testers devices got deprovisioned because of an issue with my Apple developer account. This can happen if Apple update the legal agreement and want you to reconfirm that you accept it. They did that very recently.
To see if this is the case, go to the Distribute menu in AppCenter and select Group and then Devices. There your are likely to see that they are not all Provisioned. You will then need to login to your Apple developer account and accept their updated conditions. Then go back to AppCenter and run through the automatic provisioning process again. And then you should be good again - until next time. Hope this helps, otherwise write to Microsoft with the chat function in AppCenter and they will respond rather quickly.

**Comment by omatrot** [comment]
I have the same problem today with an iPhone 8 Plus running iOS 14.3.
Is there a way to see logs on server side ?

**Comment by omatrot** [comment]
I have solved it by clearing the browser cache from the Settings app.

**Comment by ozanmanav** [comment]
Same here with iphone 11. Any Fix ?

**Comment by eliezedeck** [comment]
Same problem, iOS 14.4 on iPhone 12 Pro Max emulator.

**Comment by oprogramador** [comment]
Same problem, iPhone SE (2020), iOS 14.4.2, using a private tab in Safari doesn't help.

**Comment by s-hocking** [comment]
Encountering this same problem with an iPhone XR, iOS 14.6

**Comment by tomduncalf** [comment]
We ended up migrating away from AppCenter on iOS and instead just push our builds directly to TestFlight (we build on Azure Pipelines, so this is using the Azure App Store extension). 

Feels like there are so many moving parts to iOS App Center and I was constantly battling random issues like this, expired tokens etc. With TestFlight, you remove a lot of those moving pieces and can authenticate for uploading to it with an api token that never expires.

**Comment by oprogramador** [comment]
I missed being provisioned. If I remember correctly, somebody needed to provision me in the App Center settings, then another build was needed to be published and then I was able to download/install app from the App Center on my iPhone.

**Comment by opeyeminomy** [comment]
any fix on this ?

**Comment by GNUGradyn** [comment]
Still experiencing this. iOS 15.5 on iPhone 8 plus

**Comment by collin96** [comment]
There is a fix for this issue as mentioned ealier. Delete the safari cache, history and website data from the settings. Close all tabs and the issue should be solved. It worked for me anyway.
